esp-hal-common/src/lib.rs

```diff
@@ -162,7 +162,10 @@ pub mod trapframe {
 mod soc;
 
 #[no_mangle]
-extern "C" fn EspDefaultHandler(_level: u32, _interrupt: peripherals::Interrupt) {}
+extern "C" fn EspDefaultHandler(_level: u32, _interrupt: peripherals::Interrupt) {
+    #[cfg(feature = "log")]
+    log::warn!("Unhandled level {} interrupt: {:?}", _level, _interrupt);
```

Comment:
why is this log only?

Reply:
Oops, forget we have defmt support now. Fixed.

Reply:
yeah I'm just not sure Format is impl'd for Interrupt yet :D I'll add it to the PACs if this fails

Reply:
cc https://github.com/esp-rs/esp-pacs/pull/161 the feature will need to be enabled for the PACs. Why CI doesn't fail is beyond me.